bin/all_sky_search/pycbc_upload_single_event_to_gracedb

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
```

Comment:
I should probably say that this code would be run after the workflow has finished, and the box is opened.

We would use something like:
```
pycbc_upload_single_event_to_gracedb \
  --xml-file-for-upload \
    $upload_data_dir/H1L1V1-GENERATE_XML_FULL_DATA_UPLOAD_3-1186740100-3400.xml \
  --snr-timeseries-plot \
    $upload_data_dir/H1L1V1-GENERATE_XML_FULL_DATA_UPLOAD_3_SNR-1186740100-3400.png \
  --asd-plot \
    $upload_data_dir/H1L1V1-GENERATE_XML_FULL_DATA_UPLOAD_3_PSD-1186740100-3400.png \
  --skymap-plot \
    $upload_data_dir/H1L1V1-SKYMAP_PLOT_FULL_DATA_UPLOAD_3-1186740100-3400.png \
  --skymap-fits-file \
    $upload_data_dir/H1L1V1-BAYESTAR_FULL_DATA_UPLOAD_3-1186740100-3400.fits \
  --labels \
    GRACEDB_LABEL
```

With the upload number (3 here) being looped over and timestamp updated as appropriate